### PR TITLE
Creation of a setup.py

### PR DESCRIPTION
**Pull request by Lvl4Sword**

### CODE REVIEW

**Comment by PeterSurda** [comment]
I had to merge the language first, please rebase/resign. It's a good start so I'll merge it and give you a tip, but it's not adequate, I'm working on an improved version and commit that on top of your commit. Some of the dependencies (hashlib) are  part of python, some (pyqt) aren't supported by setuptools and so on. But I got it mostly working.

**Comment by PeterSurda** [comment]
Ok I got it working. Please rebase/resign within a day, otherwise I won't have time to delay my commit.

**Comment by g1itch** [comment]
You probably forgot a dependency on pygobject and numpy.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        "Programming Language :: Python :: 2.7.6",
+        "Programming Language :: Python :: 2.7.7",
+        "Programming Language :: Python :: 2.7.8",
+        "Programming Language :: Python :: 2.7.9",
+        "Programming Language :: Python :: 2.7.10",
+        "Programming Language :: Python :: 2.7.11",
+        "Programming Language :: Python :: 2.7.12",
+        "Programming Language :: Python :: 2.7.13",
+    ],
+    packages=find_packages(include=['src'])
```
Comment (g1itch) [inline setup.py]: missing coma
and also this installs package 'src'

**Comment by PeterSurda** [comment]
I committed my own, more advanced, version then.

**Comment by g1itch** [comment]
Maybe you can close also #171

**Pull request closed without merging**